app/move/steps/create.js

```diff
@@ -157,9 +171,11 @@ module.exports = {
       'gender_additional_information',
     ],
   },
+  // OCA journey
   '/move-date-range': {
-    ...moveDateStep,
+    pageTitle: 'moves::steps.move_date.heading',
```

Comment:
Any reason to not just keep as it was? This property is the same on the shared step.

Reply:
Basically the whole `moveDateStep` is almost entirely composed of `next` conditions, that we need to override here anyway. All of a sudden it seemed like a booby trap, importing a largish object almost entirely overridden. I think this way is clearer and cleaner.

Reply:
I was thinking more if something was added to the shared step then it wouldn't be shared any longer. And now we have created a shared step variable which is only used by one step, which voids its "sharedness".

I'd say either:

- remove the `moveDateStep` variable and add `pageTitle` on both steps separately, and add the `next` from `moveDateStep` to the `/move-date` step
- leave it as it is